llama-index-packs/llama-index-packs-cohere-citation-chat/pyproject.toml

```diff
@@ -0,0 +1,65 @@
+[build-system]
+build-backend = "poetry.core.masonry.api"
+requires = ["poetry-core"]
+
+[tool.codespell]
+check-filenames = true
+check-hidden = true
+skip = "*.csv,*.html,*.json,*.jsonl,*.pdf,*.txt,*.ipynb"
+
+[tool.llamahub]
+contains_example = false
+import_path = "llama_index.packs.cohere_citation_chat"
+
+[tool.llamahub.class_authors]
+CohereCitationChatEnginePack = "EugeneLightsOn"
+
+[tool.mypy]
+disallow_untyped_defs = true
+exclude = ["_static", "build", "examples", "notebooks", "venv"]
+ignore_missing_imports = true
+python_version = "3.8"
+
+[tool.poetry]
+authors = ["Your Name <you@example.com>"]
+description = "llama-index packs cohere_citation_chat integration"
+exclude = ["**/BUILD"]
+keywords = ["chat", "citation", "cite", "cohere", "engine", "index"]
+license = "MIT"
+maintainers = ["EugeneLightsOn"]
+name = "llama-index-packs-cohere-citation-chat"
+readme = "README.md"
+version = "0.1.3"
+
+[tool.poetry.dependencies]
+python = ">=3.8.1,<4.0"
+llama-index-core = "^0.10.1"
+thefuzz = "^0.22.1"
```

Comment:
Are these dependencies correct? Probably you can remove thefuzz and add llama-index-llms-cohere and llama-index-embeddings-cohere (not sure if there are other deps beyond that)

Reply:
Also done